draft-ietf-quic-http.md

```diff
@@ -1638,16 +1645,27 @@ H3_FRAME_UNEXPECTED.
 
 # Error Handling {#errors}
 
-QUIC allows the application to abruptly terminate (reset) individual streams or
-the entire connection; see Sections 2.4 and 5.3 of {{QUIC-TRANSPORT}}.  These
-are referred to as "stream errors" or "connection errors" (see Section 11 of
-{{QUIC-TRANSPORT}}) and have associated error codes, but do not necessarily
-indicate a problem with the connection or either implementation.  For example, a
-stream can be reset if the requested resource is no longer needed.
+When a stream cannot be completed successfully, QUIC allows the application to
+abruptly terminate (reset) that stream and communicate a reason; see Section 2.4
+of {{QUIC-TRANSPORT}}. This is referred to as a "stream error."  Stream errors
```

Comment:
```suggestion
of {{QUIC-TRANSPORT}}. This is referred to as a "stream error." An HTTP/3 implementation can deciding to close a QUIC stream and communicate the reason using an error code as defined in Section 8.1 of this document. Stream errors
```
I would like the text to be very explicit about where/how the error code comes in, i.e., it is the reason being communicated by the application, which, in this case, is the HTTP/3 implementation.